aten/src/ATen/native/cuda/Blas.cpp

```diff
@@ -268,9 +268,9 @@ static bool getDisableAddmmCudaLt() {
 #ifdef USE_ROCM
 static bool isSupportedHipLtROCmArch(int index) {
     static const std::vector<std::string> archs = {
-        "gfx90a", "gfx942",
+        "gfx90a", "gfx942"
 #if ROCM_VERSION >= 60300
-        "gfx1100", "gfx1101", "gfx1200", "gfx1201",
+        , "gfx1200", "gfx1201"
```

Comment:
Please add a comma at the end.

, "gfx1200", "gfx1201",

Also I believe Navi2x support needs to be ketp, so I suggest to keep the whole code.